tests/lightning_tensor/test_lightning_tensor.py

```diff
@@ -157,3 +158,88 @@ def test_execute_and_compute_vjp(self, method):
             match="The computation of vector-Jacobian product has yet to be implemented for the lightning.tensor device.",
         ):
             dev.execute_and_compute_vjp(circuits=None, cotangents=None)
+
+
+@pytest.mark.parametrize(
+    "wires,max_bond,MPS_shape",
+    [
+        (2, 128, [[2, 2], [2, 2]]),
+        (
+            8,
+            128,
+            [[2, 2], [2, 2, 4], [4, 2, 8], [8, 2, 16], [16, 2, 8], [8, 2, 4], [4, 2, 2], [2, 2]],
+        ),
+        (8, 8, [[2, 2], [2, 2, 4], [4, 2, 8], [8, 2, 8], [8, 2, 8], [8, 2, 4], [4, 2, 2], [2, 2]]),
+        (15, 2, [[2, 2]] + [[2, 2, 2] for _ in range(13)] + [[2, 2]]),
+    ],
+)
+def test_MPSPrep_check_pass(wires, max_bond, MPS_shape):
+    """Test the correct behavior regarding MPS shape of MPSPrep."""
+    MPS = [np.zeros(i) for i in MPS_shape]
+    dev = LightningTensor(wires=wires, method="mps", max_bond_dim=max_bond)
+    dev_wires = dev.wires.tolist()
+
+    def circuit(MPS):
+        qml.MPSPrep(mps=MPS, wires=dev_wires)
+        return qml.state()
+
+    qnode_ltensor = qml.QNode(circuit, dev)
+
+    try:
+        _ = qnode_ltensor(MPS)
+    except Exception as excinfo:
```

Comment:
why do we need this try block? wouldn't the test just fail if this fails?

Reply:
Yes I thought the same but I felt the test was too empty :sweat_smile:  I will remove the try block :smile: